pages/community/styleguide/aboutguide.md

```diff
@@ -0,0 +1,35 @@
+# AsyncAPI Style Guide
+
+## Introduction
+Welcome to the AsyncAPI Style Guide. This page serves as a guideline to contributors on how to write, structure, and format content/documentation.
+
+These guidelines help us in maintaining language, tone, and voice throughout the website. It also makes it easier for multiple contributors to work together on a single doc while ensuring consistency, accuracy, and clarity. 
+
+## Writing Style
+Below are some recommendations to consider when writing your pages.
+
+### Have your target audience in mind
+Before you begin planning and constructing your content, it is important to know who you are writing for and how the readers will benefit from your content.
+This will help you in strategizing and simplifying your writing process.
+
+### Research
+Before creating your first draft, it is beneficial to research similar resources and use them as references. Avoid plagiarism or copying and pasting from other websites unless the content already exists on our site.
+
+### Follow the writing principles
+When writing our content or documentation, we aim to achieve the following:
+- **Clear** - It is very important that your writing is as clear as possible. Avoid using jargon or ambiguous words. 
+Explain any technical terms in simple and easy-to-understand words while considering your target audience. Additionally, keep your sentences short and sweet.
+- **Concise** - When conveying information, it is crucial to be concise. This reduces redundancy and caters to the needs of the reader while providing value.
+- **Consistent** - Maintaining consistency makes it easy for multiple collaborators to work together efficiently. This ensures professionalism while making it easy to implement updates and changes.
+- **Friendly** - Your content/documentation should be relatable. Always use an active voice instead of a passive tone when writing. Moreover, adding some visual graphics and real-life examples helps the reader better understand the information more practically.
+
+- **Accurate** - Lastly, your content/documentation must be accurate and precise. The information provided should be well-researched, appropriate, and not misleading. Readers rely on our content/documentation to solve their needs while saving them time doing unnecessary research.
+
+
+### Prune and polish
+At this point, it is very important to check grammar, punctuation, and spelling. View your work with a constructive eye while putting yourself in the shoes of the reader. Thoroughly proofread your work and make use of tools such as Grammarly to help polish up your work.
+
+### Ask for feedback
+There's a whole community of contributors who are willing to support you, so don't be shy to ask for feedback. You'll probably go through a lot of rounds of editing and proofreading before you can finally get your docs merged.
```

Comment:
```suggestion
A whole community of contributors is willing to support you via detailed feedback. All documentation pull requests go through several rewriting, editing, and proofreading rounds before they're ready to merge.
```

You made it sound like getting feedback and going through review rounds was an OPTION 😈